packages/calypso-analytics/src/page-view-params.ts

```diff
@@ -0,0 +1,36 @@
+// we use this variable to track URL paths submitted to analytics.pageView.record
```

Comment:
haha yep now the git blame will lead back to me 😂 

"but it's not the cleanest code - sorry" 
~ Eru Penkman (Approved by @sirreal)

I basically just tested in the browser that it produces the same behavior as it previously did and moved it all into it's own file.

Reply:
I see now, this comment was just copy-pasted from `lib/analytics`. Not sure what the nasty coupling is that it's refering to.

`analytics.pageView.record` and `analytics.pageLoading.record` are both in in `lib/analytics`, so maybe we should just leave this code there? Can we just reimplement the parts that are needed for `recordTracksPageViewWithPageParams` to work as expected?

Reply:
Those comments seem to be incorrect/out of date. There is no `pageLoading.record` function, it isn't even in the git history, I checked out master and couldn't find any other references to the`mostRecentUrlPath` or `pathCounter` variables (appart from `analytics.timing.record`).

There is a coupling to `analytics.timing.record` because `analytics.timing.record` does re-use `mostRecentUrlPath` to submit its events. I think that's not ideal but it does it via the getMostRecentUrlPath function so at least it is explicit.

I updated the comments to reflect how the variables are being used.

Reply:
Thanks. The safest thing is probably to leave it as it is for now. If there's someone that "owns" `analytics.timing.record` then we could ask what the GA timings are being used for and if there's a cleaner way we could implement it. Maybe someone in @Automattic/team-calypso knows who's using the GA timing measurements? Seems like they're related to a config flag called `perfmon`.

Lets not do any of that in this PR though.

Reply:
ahh do you mean to leave the PR as it is now or to leave this page_view params code in `lib/analytics/index.js`? 

It looks like @gravityrail wrote these comments about tracking `last_pageview_path_with_count` and `this_pageview_path_with_count` originally in 2016. I wonder if he remembers?